packages/protocol/contracts/bridge/BridgeErrors.sol

```diff
@@ -73,4 +73,36 @@ abstract contract BridgeErrors {
 
     /// Emitted when a signal of zero is used
     error B_ZERO_SIGNAL();
+
+    // NFTVault related errors
+
+    /// Emitted when a canonical token counterpart is not found for the bridged one
+    error NFTVAULT_CANONICAL_TOKEN_NOT_FOUND();
+
+    /// Emitted when opts.amount is 0
+    error NFTVAULT_INVALID_AMOUNT();
```

Comment:
Bridge is independent from NFT/ERC20 vaults, We should move all Vault errors to vaults.